ros/src/util/packages/runtime_manager/scripts/sensing.yaml

```diff
@@ -444,7 +448,54 @@ params :
         dash        : ''
         delim       : ':='
         only_enable : True
-
+  - name  : ground_filter
+    topic : /config/ground_filter
+    msg   : ConfigGroundFilter
+    vars  :
+    - name  : sensor_model
+      desc  : sensor_model desc sample
+      label : Sensor Model
+      kind  : radio_box
+      choices: [ '64', '32', '16' ]
+      descs  : [ 'HDL-64e', 'HDL-32e', 'VLP-16' ]
+      choices_type: str
+      v     : '32'
+    - name  : sensor_height
+      desc  : sensor_height desc sample
+      label : Sensor Height
+      min   : -3.0
+      max   : 3.0
+      v     : 1.72
+    - name  : max_slope
+      desc  : max_slope desc sample
+      label : Max Slope
+      min   : 0.0
+      max   : 40.0
+      v     : 20.0
+    - name  : point_distance
+      desc  : point_distance desc sample
```

Comment:
Please add some description about these params

Reply:
Thx for review. Actually so, few developers write that description field ...